install/resources/settings.xml

```diff
@@ -876,6 +876,30 @@ min=0]]></optionscode>
 			<settingvalue><![CDATA[0]]></settingvalue>
 			<isdefault>1</isdefault>
 		</setting>
+		<setting name="threadviewcountexcludespiders">
+			<title>Exclude views by spiders in thread view count?</title>
+			<description><![CDATA[Here you can exclude views by spiders from the thread view counter.]]></description>
```

Comment:
```suggestion
		<setting name="threadviews_countspiders">
			<title>Increment Thread View Count on Spider Visits</title>
			<description><![CDATA[Turn this setting On to increment the thread view count on spiders visits.]]></description>
```

Take this how as guide for the rest. Note the wording as this setting (as per suggestion) should be the one that is Off by default.

Reply:
Done!